src/validate.rs

```diff
@@ -921,6 +922,24 @@ fn validate_member_roles(data: &Data, errors: &mut Vec<String>) {
     );
 }
 
+/// Ensure homepage URLs on GitHub are owned and managed by the Rust project
+fn validate_homepage_urls(data: &Data, errors: &mut Vec<String>) {
+    let allowed_domains: Vec<Regex> = data.config().allowed_domains().iter().map(|domain| Regex::new(domain).unwrap()).collect();
+
+    wrapper(data.repos(), errors, |repo, errors| {
+        if let Some(homepage) = &repo.homepage {
+            if !allowed_domains.iter().any(|domain| domain.is_match(homepage)) {
+                errors.push(format!(
+                    "homepage URL for {}/{} is not on an allowed domain: {}",
```

Comment:
Maybe this could print the list of allowed domains to make debugging faster?

Reply:
Hm I'm not sure if that is helpful in the output. What I haven't done yet is document the allowlist in the schema and point to the configuration file. I think that should hopefully be enough for folks to quickly find the list.

Reply:
Ok, fair enough, if it's documented than that should be enough.